spec/controllers/pages_controller_spec.rb

```diff
@@ -24,4 +24,21 @@ def index
       expect(response).to render_template(layout: false)
     end
   end
+
+  describe 'content expiry' do
```

Comment:
feels kinda random to be to include these here, since `PagesController` inherits this behavior, but so do all other controllers -- include in a spec file for `ApplicationController` instead?